test/src/main/scala/scalarules/test/twitter_scrooge/thrift/thrift2/thrift3/BUILD

```diff
@@ -9,7 +9,7 @@ thrift_library(
 
 scrooge_scala_import(
     name = "thrift3_import",
-    scala_jars = ["thrift3_scrooge.jar"],
+    scala_jars = ["//test/src/main/scala/scalarules/test/twitter_scrooge:scrooge3"],
```

Comment:
I'd like a comment (in the code) here about what's going on. Just by looking at the test is very hard to understand what's going on and how is it being tested.

Reply:
kk, hope the comment I added is useful
thx for your patience too ;D